src/common/options.cc

```diff
@@ -7121,6 +7121,24 @@ std::vector<Option> get_rgw_options() {
     .set_description("mclock limit for metadata requests")
     .add_see_also("rgw_dmclock_metadata_res")
     .add_see_also("rgw_dmclock_metadata_wgt"),
+
+    Option("rgw_asio_thread_count", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
i really hope we can avoid adding a new configurable thread pool size

eventually i'd love to see this `io_context_pool` replace the thread pools in `AsioFrontend` and `RGWAsyncRadosProcessor`, both of which are dimensioned by `rgw_thread_pool_size`. obviously it's going to take some work to get there, especially on the `RGWAsyncRadosProcessor` side.. but in the meantime, maybe we're better off leaving this new io_context_pool for RGWRados hardcoded to 2?

that way future work doesn't have to figure out how to deprecate one of the configurables

Reply:
I'll use 2 for now and roll over to claim that one as we do Asiofaction.